packages/runner-ct/cypress/component/ResizeReporter.spec.tsx

```diff
@@ -0,0 +1,56 @@
+/// <reference types="@percy/cypress" />
+import React from 'react'
+import { mount } from '@cypress/react'
+import RunnerCt from '../../src/app/RunnerCt'
+import '@packages/runner/src/main.scss'
+import eventManager from '../../src/lib/event-manager'
+import { testSpecFile } from '../fixtures/testSpecFile'
+import { makeState, fakeConfig, getPort } from './utils'
+
+describe('RunnerCt', () => {
+  beforeEach(() => {
+    cy.viewport(1000, 500)
+  })
+
+  it('resizes reporter', () => {
+    const port = getPort(location.href)
+
+    cy.intercept('GET', `http://localhost:${port}/undefined/iframes//test.js`, {
+      statusCode: 200,
+      body: testSpecFile,
+    })
+
+    mount(
+      <RunnerCt
+        state={makeState()}
+        eventManager={eventManager}
+        config={fakeConfig}
+      />,
+    )
+
+    // select spec.
+    cy.get('[data-item="/test.js"').click()
+
+    cy.get('.size-container').then(($el) => {
+      const style = $el.attr('style')
+      // extract inline transform value.
+      const [, initialScale] = style.match(/transform.\s(.+);/)
+
+      // resize reporter
+      cy.get('.Resizer.vertical').eq(2)
+      .trigger('mousedown', { eventConstructor: 'MouseEvent' })
+      .trigger('mousemove', -200, -200, { eventConstructor: 'MouseEvent', force: true })
+      .trigger('mouseup')
+
+      // AUT scale should be different.
+      cy.get('.size-container').then(($el) => {
+        const newStyle = $el.attr('style')
+        const [, newScale] = newStyle.match(/transform.\s(.+);/)
+
+        expect(newScale).not.to.eq(initialScale)
```

Comment:
Can we check for the actual expected scale (or at least if the values fall within a range)?

Reply:
This is a little difficult since the width of the window when running CI is different to running locally . 🤔 Any ideas?

Reply:
Can you force the window size when starting (I know you _can_, but is there a mechanism in Cypress to do so)?

Reply:
You could also figure out the ratio given the starting window size, but that is probably way to much trouble